releasenotes/notes/tracer-toplevel-a1600adc84be9039.yaml

```diff
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    Set required header to indicate top level span computation is done in the
+    client. This fixes an issue where spans were erroneously being marked as
+    top level when partial flushing or in certain async applications.
```

Comment:
We should spell out the impact in the product (operation name dropdown and stats quota limits).